setup.py

```diff
@@ -114,6 +104,11 @@ def readfile(fname):
     import wheelhouse_uploader.cmd
     cmdclass.update(vars(wheelhouse_uploader.cmd))
 
+python_2_6_backports = ''
```

Comment:
Question was, what does an empty string imply in `install_requires`. Ignored? Warning? Exception?